### ISSUE TITLE
Build or publish script seems to be failing

### ISSUE THREAD

**Issue by ragrum** [rollup/rollup-plugin-strip#1]
First, thank you so much for rollup @Rich-Harris !

I just installed this plugin and got this in my package.json:

```
"rollup-plugin-strip": "^1.0.2"
```

And this in my ```node_modules``` folder:

```
$ ll node_modules/rollup-plugin-strip/
total 16
-rw-r--r-- 1 1000 staff  182 Jun  9 15:22 CHANGELOG.md
drwxr-xr-x 1 1000 staff  136 Jun  9 23:04 node_modules
-rw-r--r-- 1 1000 staff 2816 Jun  9 23:03 package.json
-rw-r--r-- 1 1000 staff  744 Jun  3 15:25 README.md
-rw-r--r-- 1 1000 staff  108 Jun  3 13:35 rollup.config.js
drwxr-xr-x 1 1000 staff  102 Jun  9 15:22 src
drwxr-xr-x 1 1000 staff  136 Jun  9 23:03 test
```

No ```dist/``` folder. So, I guess that's why I got the last message when trying to use it in my _rollup.config.js_ file:

```
Cannot find module 'rollup-plugin-strip'
Error: Cannot find module 'rollup-plugin-strip'
    at Function.Module._resolveFilename (module.js:339:15)
```

It seems the build and publish package is not working properly.

**Comment by eventualbuddha** [comment]
Yep, the `dist` folder is not in the `files` entry of `package.json`. Nor is there a `jsnext:main` entry. I'll fix them in a minute.

**Comment by Rich-Harris** [comment]
d'oh! released 1.0.3, thanks
